main.py

```diff
@@ -507,6 +508,42 @@ async def set_ttl_override(value: bool):
         mi_connection.inject(channel, TTLOverride.en.value, value)
 
 
+@app.post("/dac/voltage/")
+async def set_dac_voltage(device: str, channel: int, value: float):
+    """Sets the voltage of the given DAC channel.
+    
+    Args:
+        device: The DAC device name described in device_db.py.
+        channel: The DAC channel number. For Zotino, there are 32 channels, from 0 to 31.
+        value: The voltage to set. For Zotino, the valid range is from -10V to +10V.
+    """
+    class_name = "SetDACVoltage"
+    content = f"""
+from artiq.experiment import *
+
+class {class_name}(EnvExperiment):
+    def build(self):
+        self.setattr_device("core")
+        self.dac = self.get_device({device})
+
+    @kernel
+    def run(self):
+        self.core.reset()
+        self.dac.init()
+        delay(200*us)
```

Comment:
What is this for? Is it just copied from artiq codes?

Reply:
Unless inserting a delay, it cause an underflow exception.
So I found out the test code in ARTIQ GitHub and ARTIQ also inserts a delay in [here](https://github.com/m-labs/artiq/blob/master/artiq/frontend/artiq_sinara_tester.py#L494).